### ISSUE TITLE
SGDET_on_cutom_images

### ISSUE THREAD

**Issue by keeganNull** [Maelic/SGG-Benchmark#32]
![image](https://github.com/user-attachments/assets/2de011b2-2a52-40a4-b047-0f4cec337456)

**Comment by keeganNull** [comment]
how i import the model, config
![image](https://github.com/user-attachments/assets/51098025-ec87-4661-a81e-eaad85022ce9)
the model was downloaded from your oneDrive

**Comment by Maelic** [comment]
Hi, you have to put the config.yml and the .pth file under a new  ```./checkpoints/VG150-penet-yolov8m/``` directory at the root of the codebase otherwise it won't work. 
Alternatively, you can change the variable ```PATHS_CATALOG``` in your config.yml file to the absolute path of the file in your system (something like ```THE_PATH_YOU_INSTALL_THE_CODEBASE/SGG-Benchmark/sgg_benchmark/config/paths_catalog.py```)

**Comment by keeganNull** [comment]
I did as you told, but it still didn't work. After check, I found out it caused by the config by paths_catalog.py.
![image](https://github.com/user-attachments/assets/05fba3b5-c84e-4d85-8fdf-2ba43ae531ed)
And I'm new of SGG, I want to transfer random image to scene graph. Can you tell me what I need to do. thanks.

**Comment by keeganNull** [comment]
After trying, I found out  that the weight download from MODEL_ZOO does not match the size of model predefined.
**size mismatch for roi_heads.relation.union_feature_extractor.feature_extractor.pooler.reduce_channel.0.weight: copying a param with shape torch.Size([192, 576, 3, 3]) from checkpoint, the shape in current model is torch.Size([192, 192, 3, 3]).**
It's my problem, or I need train the model on my own.

**Comment by Maelic** [comment]
Yes I am sorry but the weights in model_zoo are probably outdated. I will remove them to not confuse other people. I may retrain the models and put new weights instead when I have time.
In the meantime, you can train your own model with the codebase!

**Comment by keeganNull** [comment]
> Yes I am sorry but the weights in model_zoo are probably outdated. I will remove them to not confuse other people. I may retrain the models and put new weights instead when I have time. In the meantime, you can train your own model with the codebase!
It still worked after I change some parameters in the code.Thanks for your code. Btw, I found out that the detection result is not stable, which means yolov8 can't detect all the people.
